pages/docs/concepts/message.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Message
+weight: 5
+---
+
+## What is a message?
+A `message` is a method of communication used to transmit or exchange information from a sender to the receiver through the use of `channels`. A single `message` can be consumed by multiple independent receivers, and can be also defined as an _event_, _command_, or _request_. The sender includes a payload containing data that needs to be processed and serialized into a format appropriate for the receiver to process.
+
+
+``` mermaid
+graph LR
+subgraph one[channel]
+b([message])
+end
+
+A{{sender application}} --> b --> C{{receiver application}}
```

Comment:
lgtm! I had doubts if we should write `application` but I think it is fine as it corresponds with AsyncAPI goal, that is for applications